src/lib/y2storage/blk_device.rb

```diff
@@ -320,6 +320,13 @@ def direct_md
       descendants.detect { |dev| dev.is?(:md) && dev.devices.include?(self) }
     end
 
+    # Whether the device forms part of an LVM or MD RAID
+    #
+    # @return [Boolean] true if it forms part of an LVM or MD RAID; false otherwise.
```

Comment:
This looks like the over-explained pull request, not only to translators. :wink: